notebooks/Text_Extensions_for_Pandas_Overview.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<p>convienent --&gt; convenient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>